cfn-resources/data-lake-pipeline/cmd/resource/resource.go

```diff
@@ -91,13 +89,29 @@ func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler
 }
 
 func generateDataLakeIntegrationPipeline(currentModel *Model) *admin.DataLakeIngestionPipeline {
+	partitionFieldsArr := make([]admin.DataLakePipelinesPartitionField, len(currentModel.Sink.PartitionFields))
+	for i, partitionField := range currentModel.Sink.PartitionFields {
+		partitionFieldsArr[i] = admin.DataLakePipelinesPartitionField{
+			FieldName: *partitionField.FieldName,
```

Comment:
can Get functions in admin lie  partitionField.GetFieldName() be used?

Reply:
`partitionField` in this case is the CFN model so Get functions are not available, however adjusted so that we do not access the pointer value directly for this case.